src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -6008,15 +6036,15 @@ def create_vnet_peering(cmd, resource_group_name, virtual_network_name, virtual_
         use_remote_gateways=use_remote_gateways)
     aux_subscription = parse_resource_id(remote_virtual_network)['subscription']
     ncf = network_client_factory(cmd.cli_ctx, aux_subscriptions=[aux_subscription])
```

Comment:
`aux_subscriptions` will stop working for Track 2 SDKs due to SDK limitation https://github.com/Azure/azure-sdk-for-python/issues/8313.